packages/fuels/package.json

```
@@ -55,6 +55,7 @@
   },
   "scripts": {
     "build": "tsup --dts",
-    "build:watch": "tsup --dts --watch"
+    "build:watch": "tsup --dts --watch",
+    "prepublishOnly": "cp ../../README.md ./dist/README.md"
```

As mentioned [here](https://github.com/FuelLabs/fuels-ts/pull/692#discussion_r1053625999), I believe the file should be copied to the root dir, although I'm unsure if you'll be able to test the `prepublishOnly` step locally with `pnpm pack`.

```suggestion
    "prepublishOnly": "cp ../../README.md ./README.md"
```

---

My bad. I had to delete the current `README.md` symlink, and then the `pnpm pack` command worked as expected.